src/main/java/edu/tamu/scholars/middleware/export/controller/IndividualExportController.java

```diff
@@ -54,16 +56,68 @@ public ResponseEntity<StreamingResponseBody> export(
 
     @Override
     public IndividualModel process(IndividualModel resource) {
+        Individual individual = resource.getContent();
+        if (individual != null) {
+            if (individual.getProxy().equals(Person.class.getSimpleName())) {
+                addResource(resource, new ResourceLink(individual, "docx", "Single Page Bio", "Individual single page bio export"));
```

Comment:
This will change as these resource links are defined by available export views for display views. As these augments all individual responses I prefer the values be static and bound to the persisted entities. Which will require a bit of design to afford dynamism.